include/RAJA/DepGraphNode.hxx

```diff
@@ -92,18 +92,35 @@ public:
   DepGraphNode()
       : m_num_dep_tasks(0), m_semaphore_reload_value(0), m_semaphore_value(0)
   {
-    posix_memalign((void**)(&m_semaphore_value),
-                   _SemaphoreValueAlign_,
-                   sizeof(int));
-    *m_semaphore_value = 0;
+#ifdef _WIN32
+#if USE_ALIGNED
```

Comment:
@robinson96 what controls the USE_ALIGNED macro definition; i.e., did you add it to RAJA's config.hxx